op-challenger/README.md

```diff
@@ -34,9 +34,10 @@ Run the `op-challenger` with:
 ```shell
 DISPUTE_GAME_FACTORY=$(jq -r .DisputeGameFactoryProxy .devnet/addresses.json)
 ./op-challenger/bin/op-challenger \
-  --trace-type cannon \
+  --trace-type cannon,fast \
```

Comment:
The `fast` trace type is only used for to make withdrawals fast on devnet, so I don't think it makes sense to run a second challenger with fast types enabled - the op-challenger instance run as part of the devnet will take care of those games. Should remove the `--trace-type` option and just use the default.

Reply:
The README is a bit confusing then, because beginning of the README describes how to start a devnet.
<img width="592" alt="image" src="https://github.com/user-attachments/assets/ee1bcd9e-cd95-416e-ad21-ddab4b3041d7">

Why run with default (cannon) though? since the op-challenger in devnet runs with cannon and fast already.